doc/changelog.md

```diff
@@ -4,6 +4,8 @@
 
 * Adds support for interpreting control flow operations.
   [#31](https://github.com/PennyLaneAI/catalyst/pull/31)
+* Check demos regularly in the GH action.
+  [#37](https://github.com/PennyLaneAI/catalyst/pull/37)

```

Comment:
This sort of thing doesn't need to be in the release notes, only changes that are actually part of the released package :)